routes/Posting.js

```diff
@@ -1,10 +1,17 @@
 const express = require('express')
 const router = express.Router()
-const { get, create, getPost, updatePost } = require('../controllers/posting')
+const {
+    get,
+    create,
+    getPost,
+    updatePost,
+    applicantPost
+} = require('../controllers/posting')
 
 router.route('/getPosts').get(get)
 router.route('/createPost').post(create)
 router.route('/getPost').get(getPost)
 router.route('/updatePost').patch(updatePost)
+router.route('/applicantPost').post(applicantPost)
```

Comment:
Sorry last thing, can we just change this to `apply`

It makes a bit more sense. so the endpoint would be `post/apply`